Sources/Core/Presenter/Channel/ChannelPresenter+Request.swift

```diff
@@ -44,11 +44,8 @@ extension ChannelPresenter {
             return .none
         }
         
-        if isNextPage {
-            return .reloaded(max(items.count - currentCount - 1, 0), items)
-        }
-        
-        return .reloaded((items.count - 1), items)
+        let scrollToRow = max(items.count - currentCount - 1, 0)
+        return .reloaded(scrollToRow, items)
```

Comment:
Open any channel with at least 2 pages and scroll to the first one. At the end, it will jump to the first message instead of keeping the current position.